packages/components/src/components/listitem/listitem.stories.ts

```diff
@@ -183,3 +196,22 @@ export const ListWithIconAndLabels: StoryObj = {
   `),
   ...hideAllControls(),
 };
+
+export const ListWithLongText: StoryObj = {
+  render: () => html`
+    <div role="list" style="width: 15rem">
+      <mdc-listitem label="A short text"></mdc-listitem>
+      <mdc-listitem label="A little lengthy text"></mdc-listitem>
+      <mdc-listitem
+        label="A long lengthy text with small secondary label"
+        secondary-label="a small secondary label"
+      ></mdc-listitem>
+      <mdc-listitem
+        label="A small primary label"
+        secondary-label="a lengthy secondary label with a small label"
+      ></mdc-listitem>
+      <mdc-listitem label="A short text"></mdc-listitem>
+    </div>
```

Comment:
```suggestion
<div role="list" style="width: 15rem">
      <mdc-listitem label="A short text"></mdc-listitem>
      <mdc-listitem label="A little lengthy text"></mdc-listitem>
      <mdc-listitem
        label="A long lengthy text with small secondary label"
        secondary-label="a small secondary label"
        tooltip-text="A long lengthy text with small secondary label"
        tooltip-placement="bottom"
      ></mdc-listitem>
      <mdc-listitem
        label="A small primary label"
        secondary-label="a lengthy secondary label with a small label"
        tooltip-text="A lengthy secondary label with a small label"
      ></mdc-listitem>
      <mdc-listitem label="A short text"></mdc-listitem>
    </div>
```

We would need to add the tooltip-text for this example story.

After adding this, if we test the storybook, the tooltip placement of "bottom" is not working as expected.